tern/analyze/docker/container.py

```diff
@@ -179,8 +179,8 @@ def extract_image_metadata(image_tag_string):
             os.remove(placeholder)
         if not os.listdir(temp_path):
             raise IOError('Unable to untar Docker image')
-    except docker.errors.APIError:  # pylint: disable=try-except-raise
-        raise
+    except docker.errors.APIError as e:
+        raise docker.errors.APIError(e)
```

Comment:
`load_base_image()` and `load_full_image()` in `report.py` both call `load_image()` which calls `extract_image_metadata()` where we raise this error. Do we want to catch the `APIError` error specifically in `load_base_image()` and `load_full_image`?

Reply:
It's not related to this fix, but I can submit a PR to add that change.

Reply:
Thanks! I think it makes sense to catch it along with the others. In the meantime, I'll merge this.